x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/ComputeService.java

```diff
@@ -434,7 +431,7 @@ void runCompute(CancellableTask task, ComputeContext context, PhysicalPlan plan,
         driverRunner.executeDrivers(
             task,
             drivers,
-            transportService.getThreadPool().executor(ESQL_WORKER_THREAD_POOL_NAME),
+            transportService.getThreadPool().executor(EsqlPlugin.ESQL_WORKER_THREAD_POOL_NAME),
```

Comment:
nit: maybe leave the static import for this one, so it does not appear that we are changing anything on this line?

Reply:
Yes, fixed in https://github.com/elastic/elasticsearch/pull/106050/commits/d0fe06059baea25022773d8f87742b42831de30c